### DISCUSSION TITLE
Monitor diversity of participation in the citizen science project

### DISCUSSION THREAD

**Discussion by KirstieJane** [alan-turing-institute/AutSPACEs#364] [General]
We need to think about diversity and inclusion in recruitment. In previous focus groups we’ve made some attempts to make sure the group is at least slightly diverse when it comes to ethnic minorities, gender, and parent/autistic people ratios, but our instrument has been researcher discretion. Maybe longer term we need to think through more how we might measure this, and put in the protocol strategies to proactively engage to redress disbalances? (We need to do this actively to reflect our values). This might be something worth considering when developing a future iteration of the ethics protocol.

**Comment by GeorgiaHCA** [comment]
Now that we are at a new stage of development it would be good I think to renew strategy on this

**Comment by GeorgiaHCA** [comment]
https://www.raeng.org.uk/publications/other/dilg-useful-organisations - a version of this tailored to the project could be useful, and we could begin to approach organisations for a more diverse representation
